stdlib/messaging/rabbitmq/src/main/ballerina/rabbitmq/rabbitmq_commons.bal

```diff
@@ -108,4 +111,18 @@ public type ConnectionConfiguration record {|
     int? handshakeTimeout = ();
     int? shutdownTimeout = ();
     int? heartbeat = ();
+    SecureSocket? secureSocket = ();
+|};
+
+# Provides configurations for facilitating secure connections.
+#
+# + trustStore - Configurations associated with the TrustStore.
+# + keyStore - Configurations associated with the KeyStore.
```

Comment:
```suggestion
# + keyStore - Configurations associated with the Key Store.
```

Reply:
Better to have this as one word as 'Keystore'. Key Store will give a different meaning.